criu/Makefile.packages

```diff
@@ -7,6 +7,7 @@ REQ-RPM-PKG-NAMES	+= protobuf-python
 REQ-RPM-PKG-NAMES	+= libnl3-devel
 REQ-RPM-PKG-NAMES	+= libcap-devel
 REQ-RPM-PKG-NAMES	+= $(PYTHON)-future
+REQ-RPM-PKG-NAMES	+= libbpf-devel
```

Comment:
Could you add this dependency in the Dockerfile templates (in `scripts/build/`) to enable Travis to run the tests?

Reply:
Currently, the helper functions and structures for BPF batch operations (used in this PR) are not available in 'stable' releases of libbpf-dev. For instance, Debian has the latest version of libbpf-dev (v1:0.0.9-1) only in its 'testing' and 'unstable' repositories. I see from the Dockerfile templates that only 'stable' repositories are added to the package sources list file. Therefore, the code will not compile. 

Would you like me to update the Dockerfile templates anyway?